runtime/src/bank/check_transactions.rs

```diff
@@ -184,6 +188,54 @@ impl Bank {
     }
 }
 
+/// If the message uses a durable nonce, return the pubkey of the nonce account
+fn get_durable_nonce(message: &impl SVMMessage) -> Option<&Pubkey> {
```

Comment:
Would it be possible to make this, and `get_is_signer()`, functions extension of `SVMMessage`, or at least make them live outside of `Bank` - they don't need `bank anyway?

Reply:
Moved functions and tests to a new mod inside `runtime`: `svm_message_ext`.